csrankings-9.csv

```diff
@@ -1531,6 +1531,7 @@ Yingjiu Li,University of Oregon,https://cs.uoregon.edu/profile/yingjiul,Z4ReDEMA
 Yingmin Tang,Peking University,http://www.icst.pku.edu.cn/cscl?page_id=22,NOSCHOLARPAGE
 Yingshu Li,Georgia State University,https://grid.cs.gsu.edu/yli,sgiysmUAAAAJ
 Yingwei Luo,Peking University,http://gis.pku.edu.cn/lab/people/lyw.htm,NOSCHOLARPAGE
+Yingying Chen 0001,Rutgers University,http://winlab.rutgers.edu/~yychen/,jCZaWOEAAAAJ

```

Comment:
Faculty member in ECE, not CS. Not listed here: https://www.cs.rutgers.edu/people/affiliated-faculty. Please clarify how this faculty member meets the inclusion criteria (can solely advise PhD students _in Computer Science_).

Reply:
Prof. Chen is actually with both ECE department and WINLAB, and she is also the associate director of WINLAB. WINLAB is a cooperative industry-university research center on the discipline of both computer science and computer engineering. She is also the PhD alumni of CS department of Rutgers (https://www.cs.rutgers.edu/academics/graduate/ph-d-program/ph-d-alumni). Most of her papers are published on mobile computing and computer security conferences and journals (including MobiCom, MobiSys, SenSys, CCS, etc.). Hence, within WINLAB, she is able to solely advise PhD students of both ECE and CS.

Reply:
Please point to a page that indicates that WINLAB faculty can solely advise PhD students in _Computer Science_ (that is, for a PhD granted by the CS department). Thanks.

Reply:
I am going to close this PR, but if you can point me to the evidence asked for above, please feel free to re-open.